backend/composer/models.py

```diff
@@ -220,7 +220,7 @@ class AnatomicalEntity(models.Model):
     """Anatomical Entity"""
 
     name = models.CharField(max_length=200, db_index=True)
-    ontology_uri = models.URLField()
+    ontology_uri = models.URLField()  # TODO: Confirm it is acceptable to have multiple anatomical entities with the same uri.
```

Comment:
I noticed that we have multiple anatomical entities for the same ontology_uri. Is that expected? When transforming the uris from neurondm into anatomical entities in sckan I'm getting the first match

Reply:
@afonsobspinto sounds good to me. In case the user would like to explicit pick sort the AE we would need to add a sort order attribute to AE. And then we would need to sort here by the sort order. 

an AE can have an alias, the alias is in Composer an AE with a different name but the same ontology_uri